src/components/calculator.js

```diff
@@ -15,54 +15,56 @@ const plusMinus = require("./icons/plus-minus-icon");
 
 function calculator(id) {
 	const buttonsFirstLine = buttonsGroup("buttonsFirstLine", [
-		button(clearAll(), "button is-large is-light m-2 is-fullwidth is-one-fifth has-text-danger", "C"), 
-		button(plusMinus(), "button is-large is-light m-2 is-fullwidth is-one-fifth has-text-warning", "+/-"),
-		button(delet(), "button is-large is-light m-2 is-fullwidth is-one-fifth has-text-warning", "D"), 
-		button(percent(), "button is-large is-light m-2 is-fullwidth is-one-fifth has-text-warning", "%")
+		button(clearAll(), "is-large is-light m-2 is-fullwidth is-one-fifth has-text-danger is-mobile", "C"), 
```

Comment:
Please just do changes related with the ticket

Reply:
i know, but when you merged the last pr i did a few changes that i forgot to commit and because was pending things i did inside the same ticket, how can i do this without do it inside a non respective ticket?

Reply:
open a new PR for those changes!

Reply:
ok i get it